File: clients/php/src/Exception/CapacityExhaustedError.php
<?php

declare(strict_types=1);

namespace Priostack\Acn\Exception;

/** {@code capacity-exhausted} — a tier/registration/store ceiling was hit. */
class CapacityExhaustedError extends AcnToolError
{
    public const OUTCOME = 'capacity-exhausted';
}

File: clients/java/src/main/java/com/priostack/acn/CapacityExhaustedError.java
package com.priostack.acn;

import com.fasterxml.jackson.databind.JsonNode;

/** {@code capacity-exhausted} — a tier/registration/store ceiling was hit. */
public class CapacityExhaustedError extends ACNToolError {

    private static final long serialVersionUID = 1L;

    public CapacityExhaustedError(String outcome, String detail, String method, JsonNode data) {
        super(outcome, detail, method, data);
    }
}
